### PR TITLE
Added initial models

### PR DESCRIPTION
**Pull request by aaboffill**
[Creation of initial models #6 ](https://github.com/savoirfairelinux/sflvault-server/issues/6)

* Added initial models
* Added base DB configuration
* Added base admin file

### CODE REVIEW

File: vaultsrv/admin.py [hunk 1]
```diff
@@ -1,3 +1,8 @@
-# from django.contrib import admin
+from django.contrib import admin
 
-# Register your models here.
+from vaultsrv.models import (
+    Account, AccountGroup, GroupMembership, Customer, ServiceGroup, Service, ServiceGroupMembership
+)
+
+# This is no the final admin definition, it's is only to test the models
+admin.site.register((Account, AccountGroup, GroupMembership, Customer, ServiceGroup, Service, ServiceGroupMembership))
```
Comment (peakwinter) [inline vaultsrv/admin.py]: Could you break these lines into two so they fall under 80 characters per line? :)

File: vaultsrv/models/services.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+from vaultsrv.models import users
```
Comment (peakwinter) [inline vaultsrv/models/services.py:3]: Lots of lines in this file to shorten as well!

**Review by peakwinter — CHANGES_REQUESTED** [review]

**Review by peakwinter — APPROVED** [review]

**Comment by aaboffill** [comment]
@peakwinter I delimited the python code lines to 80. It's very little in my consideration :( but done! :+1:

**Comment by peakwinter** [comment]
Thanks @aaboffill !!

**Pull request merged**